ios/FluentUI/Command Bar/CommandBarItem.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+public typealias CommandBarItemGroup = [CommandBarItem]
+
+open class CommandBarItem: NSObject {
+    public init(
+        iconImage: UIImage?,
+        isEnabled: Bool = true,
+        isSelected: Bool = false,
+        isPersistSelection: Bool = true,
+        accessbilityLabel: String? = nil
+    ) {
+        self.iconImage = iconImage
+        self.isEnabled = isEnabled
+        self.isSelected = isSelected
+        self.isPersistSelection = isPersistSelection
+
+        super.init()
+
+        self.accessibilityLabel = accessbilityLabel
+    }
+
+    @available(iOS 14.0, *)
+    public init(
+        iconImage: UIImage?,
+        isEnabled: Bool = true,
+        isSelected: Bool = false,
+        isPersistSelection: Bool = true,
+        menu: UIMenu,
+        showsMenuAsPrimaryAction: Bool = false,
+        accessbilityLabel: String? = nil
+    ) {
+        self.iconImage = iconImage
+        self.isEnabled = isEnabled
+        self.isSelected = isSelected
+        self.isPersistSelection = isPersistSelection
+
+        super.init()
+
+        self.menu = menu
+        self.showsMenuAsPrimaryAction = showsMenuAsPrimaryAction
+        self.accessibilityLabel = accessbilityLabel
+    }
+
+    public var iconImage: UIImage?
+    public var isEnabled: Bool
+
+    /// If `isPersistSelection` is `true`, this value would be changed to reflect the selection state of the button. Setting this value before providing to `CommandBar` would set the initial selection state.
+    public var isSelected: Bool {
+        didSet {
+            if isSelected, !isPersistSelection {
+                isSelected = false
+            }
+        }
+    }
+
+    /// Whether the selection state is persisted. If this is set to `false`, the button would be deselected immediately after selection.
+    public var isPersistSelection: Bool
+
+    @available(iOS 14.0, *)
+    public lazy var menu: UIMenu? = nil // Only lazy property can be used with @available
```

Comment:
I am having a hard time figuring out the use case for this? I don't see any menu in iOS 14. Does anything specific action triggers this?

Reply:
Long press to show the menu. Or set `showsMenuAsPrimaryAction` to `true`, then the button would show the menu when tapped.